pages/get-started/index.tsx

```diff
@@ -43,17 +44,22 @@ const GetStarted: NextPage = () => {
           <button
             type="button"
             onClick={async () => {
-              await signIn("github", { callbackUrl: redirectTo });
+              try {
+                await signIn("github", { callbackUrl: redirectTo });
+                Fathom.trackGoal('sign-up-complete', 0);
```

Comment:
We share a sign-in and sign-up page so this would trigger every time a user signs in. Can the client be added to the settings page when the user account is being setup in the `getServerSideProps`? I've not setup one of these trackers so I really don't know.

Reply:
Will do!